src/esp/gfx/TextRenderer.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <Corrade/Containers/Containers.h>
+#include <Corrade/PluginManager/Manager.h>
+#include <Magnum/Math/Color.h>
+#include <Magnum/Math/Matrix3.h>
+#include <Magnum/Shaders/Vector.h>
+#include <Magnum/Text/AbstractFont.h>
+#include <Magnum/Text/Alignment.h>
+#include <Magnum/Text/GlyphCache.h>
+#include <Magnum/Text/Renderer.h>
+#include <string>
+#include <vector>
+
+namespace esp {
+namespace gfx {
+
+namespace Cr = Corrade;
+namespace Mn = Magnum;
+
+class TextRenderer {
+ public:
+  /**
+   * @brief Constructor
+   */
+  // aspectRatio = width / height
+  explicit TextRenderer(float viewportAspectRatio = 1.33333f,
+                        float fontSize = 20.0f,
+                        int cacheSize = 1024);
```

Comment:
Is this the cache texture size you mentioned in your previous comments? 
Or the fontSize above it?

Reply:
Can't tell what are the right values without playing with them myself :) You have two degrees of freedom here:

- size of the pre-rendered glyphs (and cache size is just large enough to contain them all, no point in having it too large ... I guess maybe even 256x256 could be enough)
- size of the text on-screen

These are orthogonal (size of one doesn't affect size of the other), for given size of the text on screen you basically need to pick large enough `fontSize` to have it rendered not too blurry but also not too aliased.

Reply:
I see. so no need to set the cacheSize so big, and whether the "text" look sharp or not depends on the `fontSize` here and the text size on screen (set later).